packages/runtime/container-runtime/api-report/container-runtime.legacy.alpha.api.md

```diff
@@ -340,7 +340,6 @@ export interface IContainerRuntimeMetadata extends ICreateContainerMetadata, IGC
 export interface IContainerRuntimeOptions {
     readonly chunkSizeInBytes?: number;
     readonly compressionOptions?: ICompressionRuntimeOptions;
-    readonly enableGroupedBatching?: boolean;
```

Comment:
this is a breaking change, so need to go through a deprecation, notify, and removal in a .10 release